### PR TITLE
Ci bring tests

### PR DESCRIPTION
**Pull request by mich-master**

### CODE REVIEW

File: init-governance.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -euo pipefail
+
 solana -v --url http://localhost:8899 --keypair artifacts/creator.keypair airdrop 100
```
Comment (s-medvedev) [inline init-governance.sh:4]: Remove `--url http://localhost:8899`. Let `solana` and `spl-token` use URL from config.

File: run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
```
Comment (s-medvedev) [inline run-tests.sh:3]: Add: `solana set config -u ${SOLANA_URL:-http://localhost:8899}`

File: init-governance.sh (hunk 1 above)
Comment (mich-master) [reply init-governance.sh]: Fixed

File: run-tests.sh (hunk 2 above)
Comment (mich-master) [reply run-tests.sh:3]: Fixed

File: solana-run-neon.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```
Comment (s-medvedev) [inline solana-run-neon.sh:1]: This file seems unneeded.

Comment (mich-master) [reply solana-run-neon.sh]: Fixed

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**